### DISCUSSION TITLE
Features you would like to work on.

### DISCUSSION THREAD

**Discussion by PranishKansakar** [PranishKansakar/Tafesa240154#3] [Polls]
Hey team,

Hope everyone's doing good! Let's decide who's taking charge of which feature for our program. We've got three options: Mortgage Calculator, Currency Calculator, and Trip Calculator. 

Please take a moment to indicate your preference for the feature you'd like to work on.

**Comment by Omonotuaga** [comment]
Good day to you Pranish, thank you for this query. I would like to express my intent to take charge in creating the trip calculator feature of our program. 

Thank you very much

Autonomo Amano

**Comment by GEONHWISONG** [comment]
I voted with Mortgage Calculator! I have a pretty confidence with hand with this feature!
